### PR TITLE
[Australian War Memorial] Enabled ruleset, added subdomains, improved style, documented issues

### PR DESCRIPTION
**Pull request by galenide**

### CODE REVIEW

**Comment by galenide** [comment]
@gloomy-ghost @jeremyn 

As far as reducing the backlog of open pull requests, this request from August of this year may be of interest.

**Comment by jeremyn** [comment]
Though I understand the frustration, I have to discourage pinging reviewers on old pull requests that haven't been reviewed yet and are not actually urgent. If everyone did that, we would be flooded.

File: src/chrome/content/rules/AustralianWarMemorial.xml [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+
 -->
-<ruleset name="Australian War Memorial" default_off='failed ruleset test'>
+<ruleset name="Australian War Memorial (partial)">
+
 	<target host="awm.gov.au" />
-	<target host="*.awm.gov.au" />
+	<target host="cas.awm.gov.au" />
+	<target host="static.awm.gov.au" />
+	<target host="www.awm.gov.au" />
```
Comment (J0WI) [inline src/chrome/content/rules/AustralianWarMemorial.xml]: See also `webmail.awm.gov.au`

**Comment by jeremyn** [comment]
@galenide Are you still interested in working on this?

**Comment by jeremyn** [comment]
I'm closing this due to lack of response from @galenide .

**Pull request closed without merging**